language/tools/move-unit-test/tests/move_unit_test_testsuite.rs

```diff
@@ -127,14 +97,14 @@ fn run_test(path: &Path) -> datatest_stable::Result<()> {
             let expected = fs::read_to_string(&exp_path)?;
             let output = String::from_utf8(buffer)?;
             if expected != output {
-                return move_lang_test_utils::error(format!(
+                anyhow!(format!(
```

Comment:
`anyhow!` should also be able to take format strings. That should make this and the one below a bit cleaner.